daprdocs/content/en/developing-applications/building-blocks/scheduler/scheduler-overview.md

```diff
@@ -0,0 +1,64 @@
+---
+type: docs
+title: "Scheduler overview"
+linkTitle: "Overview"
+weight: 1000
+description: "Overview of the scheduler API building block"
+---
+
+The Scheduler API works as an orchestrator for scheduling jobs in the future,  either at a specific time or a specific interval. The scheduler helps you with jobs like:
+
+- Scalable actor reminders
+- Scheduling any Dapr API to run at specific times or intervals, like when:
```

Comment:
This will come, but we don't have this quite yet- scheduling other Dapr APIs. Maybe for now we can say scheduling jobs in a general sense and in the following releases expanded to scheduling other Dapr APIs (which are those bullets)...

Reply:
Sounds good! I'll keep from promising anything and just stick to something like the following in line 9: 
> "...scheduling jobs, including scalable actor reminders."